example/javalib/android/2-linting/build.mill

```diff
@@ -0,0 +1,29 @@
+package build
+
+import mill._
+import mill.javalib.android.{AndroidSdkModule, AndroidLintModule}
+
+// Create and configure an Android SDK module to manage Android SDK paths and tools.
+object androidSdkModule0 extends AndroidSdkModule {
+  def buildToolsVersion = "35.0.0"
+}
+
+// Actual android application
+object app extends AndroidLintModule {
+  def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
+}
+
+/** See Also: app/AndroidManifest.xml */
+/** See Also: app/src/main/java/com/helloworld/app/MainActivity.java */
+
+/** Usage
+
+> ./mill show app.androidLint  # Display full path to the linting report in HTML
+".../out/app/androidLint.dest/report.html"
```

Comment:
We should use `cat` on the HTML report to ensure certain lint errors get picked up and properly reported